www/app/datasets/view/[dataset_id]/page.tsx

```
@@ -0,0 +1,202 @@
+                                                    </button>
+                                                </TableCell>
+                                                <TableCell>
+                                                    <p className="whitespace-pre-wrap">{image.reference_text}</p>
+                                                </TableCell>
+                                                <TableCell>
+                                                    <a href={image.url} target="_blank" rel="noopener noreferrer" className="text-blue-600 underline break-all">
+                                                        {image.url}
+                                                    </a>
+                                                </TableCell>
+                                            </TableRow>
+                                        ))}
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)